### ISSUE TITLE
How can I subscribe in Slack Github for new releases?

### ISSUE THREAD

**Issue by Aracki** [prometheus/prometheus#10939]
I'd like to be able to subscribe to this repository releases within Slack Github app.

E.g.

For https://github.com/grafana/grafana I'm getting: 
<img width="480" alt="slika" src="https://user-images.githubusercontent.com/6129986/176701992-6684e820-053d-414c-b5b3-c382cbf6f91c.png">

... but for this repository: 
<img width="571" alt="slika" src="https://user-images.githubusercontent.com/6129986/176702071-3fbd7308-e56c-4d34-af2f-3e24dc96f25f.png">

**Comment by LeviHarrison** [comment]
Yeah, so according to [this issue](https://github.com/integrations/slack/issues/454), that would require an admin installing the Slack integration on this repo. I don't see any problem with that, and I'll ask around to see if someone's interested in doing it. But I don't know if it will happen, so as a workout you could add the releases RSS feed (https://github.com/prometheus/prometheus/releases.atom) [to Slack](https://slack.com/help/articles/218688467-Add-RSS-feeds-to-Slack).

**Comment by Aracki** [comment]
Thanks @LeviHarrison for the suggestion, but unfortunately I'm getting:
> Rats, we couldn’t connect to the URL (http://prometheus//releases.atom).

when I run:
```
/feed subscribe prometheus/prometheus/releases.atom
```

**Comment by roidelapluie** [comment]
/feed subscribe https://github.com/prometheus/prometheus/releases.atom

**Comment by Aracki** [comment]
It's the same command I've already tried 🙁

**Comment by roidelapluie** [comment]
You did not specify https://github.com at the beginning

**Comment by Aracki** [comment]
oh my bad. Thank you! 👍
